### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'get_dialogs'

### ISSUE THREAD

**Issue by giorgiomess** [TelegramTools/TLMerger#3]
Hello, I'm having this issue running the source code via Python3 IDLE installed on macOS. I've replaced YOUR_API_ID_HERE and YOUR_API_HASH_HERE with my API ID and HASH, and the program works well from logging to selecting chats (solo mode) and parameters before starting merging process.
But when it starts it says two errors from Python, they seem syntax error... but I don't really now. I attach a screenshot (don't worry about red warnings, it's Python IDLE) and log.
Thanks.
<img width="569" alt="screenshot" src="https://user-images.githubusercontent.com/72503638/106290891-45ea8a00-624b-11eb-8215-8858ab68f149.png">
[TLMerger-log.log](https://github.com/TelegramTools/TLMerger/files/5894522/TLMerger-log.log)

**Comment by ferferga** [comment]
@rgiorgiotech Indeed a conditional error

Can you pull my latest changes and retry, please?

**Comment by giorgiomess** [comment]
> @rgiorgiotech Indeed a conditional error
> 
> Can you pull my latest changes and retry, please?

Ok now it says another issue when starting process:

`STARTED! Now copying the conversation in Telegram...


Processing...

INFORMATION: Each 1000 messages, a pause will happen because TLMerger needs to refresh metadata due to Telegram's restrictions.
Although the process might look like it's stuck, don't close or cancel it because you think it's stuck. Whenever an error happens, you will be informed.


You can cancel at any time using CTRL+C keyboard combination.
Traceback (most recent call last):
  File "/Users/giorgiomessina/Desktop/TLMerger-1.1/src/TLMerger.py", line 1381, in ExportMessages
    Welcmsg = SendMessageClient1(user2,
UnboundLocalError: local variable 'user2' referenced before assignment

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/giorgiomessina/Desktop/TLMerger-1.1/src/TLMerger.py", line 3603, in <module>
    ExportMessages()
  File "/Users/giorgiomessina/Desktop/TLMerger-1.1/src/TLMerger.py", line 3383, in ExportMessages
    bar.finish()
UnboundLocalError: local variable 'bar' referenced before assignment
>>> `
[TLMerger-log.log](https://github.com/TelegramTools/TLMerger/files/5894736/TLMerger-log.log)

**Comment by ferferga** [comment]
You sure you installed all the dependencies using pip?

**Comment by giorgiomess** [comment]
> You sure you installed all the dependencies using pip?

Yes of course. I've installed all the dependencies in their specific version you wrote on txt file.

**Comment by giorgiomess** [comment]
I can try non-solo mode instead... I'll let you know.

**Comment by giorgiomess** [comment]
Nothing to do. Non-solo mode isn't good for group chats! I'd like to merge two group chats but for this purpose I need the solo-mode.
I really hope you'll fix these issues!
